adt.sh

```diff
@@ -71,7 +71,7 @@ env_var "INSTANCES_CONF_DIR" "${ADT_DATA}/conf/instances"
 env_var "ETC_DIR" "${ADT_DATA}/etc"
 
 env_var "CURR_DATE" `date -u "+%Y%m%d.%H%M%S"`
-env_var "REPOS_LIST" "exoplatform:gatein-dep exodev:gatein-wci exodev:kernel exodev:core exodev:ws exodev:jcr exodev:gatein-sso exodev:gatein-pc exodev:gatein-portal exoplatform:maven-depmgt-pom exodev:docs-style exodev:platform-ui exodev:commons exodev:calendar exodev:forum exodev:wiki exodev:social exodev:ecms exodev:integration exodev:platform exoplatform:platform-public-distributions exoplatform:platform-private-trial-distributions exoplatform:platform-private-distributions"
+env_var "REPOS_LIST" "exoplatform:gatein-dep exodev:gatein-wci exodev:kernel exodev:core exodev:ws exodev:gatein-sso exodev:gatein-pc exodev:gatein-portal exoplatform:maven-depmgt-pom exodev:platform-ui exo-addons:enterprise-skin exodev:commons exodev:social exoplatform:addons-manager exoplatform:platform-public-distributions exoplatform:platform-private-distributions exodev:wiki exodev:jcr exodev:ecms exodev:calendar exodev:forum exo-addons:gamification exo-addons:wallet exo-addons:kudos exo-addons:perk-store exo-addons:cas-addon exo-addons:chat-application exo-addons:cmis-addon exo-addons:data-upgrade exo-addons:digital-workplace exo-addons:exo-es-embedded exo-addons:layout-management exo-addons:lecko exo-addons:legacy-intranet exo-addons:news exo-addons:onlyoffice exo-addons:openam-addon exo-addons:push-notifications exo-addons:remote-edit exo-addons:saml2-addon exo-addons:spnego-addon exo-addons:task exo-addons:wcm-template-pack exo-addons:web-conferencing"
```

Comment:
```suggestion
env_var "REPOS_LIST" "exoplatform:gatein-dep exodev:gatein-wci exodev:kernel exodev:core exodev:ws exodev:gatein-sso exodev:gatein-pc exodev:gatein-portal exoplatform:maven-depmgt-pom exodev:platform-ui exoplatform:enterprise-skin exodev:commons exodev:social exoplatform:addons-manager exoplatform:platform-public-distributions exoplatform:platform-private-distributions exodev:wiki exodev:jcr exodev:ecms exodev:calendar exodev:forum exo-addons:gamification exo-addons:wallet exo-addons:kudos exo-addons:perk-store exo-addons:cas-addon exo-addons:chat-application exo-addons:cmis-addon exo-addons:data-upgrade exo-addons:digital-workplace exo-addons:exo-es-embedded exo-addons:layout-management exo-addons:lecko exo-addons:legacy-intranet exo-addons:news exo-addons:onlyoffice exo-addons:openam-addon exo-addons:push-notifications exo-addons:remote-edit exo-addons:saml2-addon exo-addons:spnego-addon exo-addons:task exo-addons:wcm-template-pack exo-addons:web-conferencing"
```
The change is not very visible on the suggested change but the exo-enterprise-skin location was changed from exo-addons to exoplatform

Reply:
change applied